contracts/mocks/docs/ERC4626Fees.sol

```diff
@@ -64,27 +69,41 @@ abstract contract ERC4626Fees is ERC4626 {
         }
     }
 
-    function _entryFeeBasePoint() internal view virtual returns (uint256) {
-        return 0;
+    // === Fee configuration === ///
+
+    function _entryFeePCM() internal view virtual returns (uint256) {
+        return 0; // i.e. 1_000 for 1%
     }
 
-    function _entryFeeRecipient() internal view virtual returns (address) {
-        return address(0);
+    function _exitFeePCM() internal view virtual returns (uint256) {
+        return 0; // i.e. 1_000 for 1%
     }
 
-    function _exitFeeBasePoint() internal view virtual returns (uint256) {
-        return 0;
+    function _entryFeeRecipient() internal view virtual returns (address) {
+        return address(0); // i.e. treasury
     }
 
     function _exitFeeRecipient() internal view virtual returns (address) {
-        return address(0);
+        return address(0); // i.e. treasury
     }
 
+    // === Fee operations === ///
+
+    /// @dev Calculates fee on `assets` not including fees. Used in {IERC4626-mint} and {IERC4626-withdraw} operations.
     function _feeOnRaw(uint256 assets, uint256 feeBasePoint) private pure returns (uint256) {
+<<<<<<< Updated upstream
         return assets.mulDiv(feeBasePoint, 1e5, Math.Rounding.Ceil);
+=======
+        return assets.mulDiv(feeBasePoint, _PCM_SCALE, Math.Rounding.Up);
+>>>>>>> Stashed changes
     }
 
+    /// @dev Calculates fee on `assets` already including fees. Used in {IERC4626-deposit} and {IERC4626-redeem} operations.
     function _feeOnTotal(uint256 assets, uint256 feeBasePoint) private pure returns (uint256) {
+<<<<<<< Updated upstream
         return assets.mulDiv(feeBasePoint, feeBasePoint + 1e5, Math.Rounding.Ceil);
+=======
+        return assets.mulDiv(feeBasePoint, feeBasePoint + _PCM_SCALE, Math.Rounding.Up);
+>>>>>>> Stashed changes
```

Comment:
```suggestion
    function _feeOnRaw(uint256 assets, uint256 feeBasePoint) private pure returns (uint256) {
        return assets.mulDiv(feeBasePoint, _PCM_SCALE, Math.Rounding.Ceil);
    }

    /// @dev Calculates fee on `assets` already including fees. Used in {IERC4626-deposit} and {IERC4626-redeem} operations.
    function _feeOnTotal(uint256 assets, uint256 feeBasePoint) private pure returns (uint256) {
        return assets.mulDiv(feeBasePoint, feeBasePoint + _PCM_SCALE, Math.Rounding.Ceil);
```

Reply:
Issues of pushing just before running into a call. Thanks!